packages/client/src/fuature/forum/components/notification/Notification.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import styles from './Notification.module.scss';
+import { Alert } from './components/Alert'
+import { AlertTitle } from './components/AlertTitle'
+import { Box } from './components/Box'
+import { AlertIcon } from './components/AlertIcon/AlertIcon'
+import { AlertDesc } from './components/AlertDesc/AlertDesc'
+import { AlertButton } from './components/AlertButton/AlertButton'
+
+type NotificationProviderT = {
+  props?: React.ComponentProps<any>;
+}
+
+export const NotificationAPI: React.FC<NotificationProviderT> = () => {
+  async function notify (notificationText = "test Notification") {
+    if(!("Notification" in window)) {
+    alert("Браузер не поддерживает уведомления!")
+  } else if (Notification.permission === "granted") {
+    const notification = new Notification(notificationText);
```

Comment:
Кажется достаточно просто 
```suggestion
new Notification(notificationText);
```
так как ты не используешь ниже переменную notification.